modules/efk-logging-fluentd-external.adoc

```diff
@@ -51,10 +76,45 @@ $ oc edit configmap/fluentd -n openshift-logging
 ----
 <1> Enter the name, host, and port for your external Fluentd server.
 
+. Add the path to your certificate and private key to the `secure-forward.conf` section:
++
+[source,yaml]
+----
+  secure-forward.conf: |
+    # <store>
+    # @type secure_forward
+
+    # self_hostname ${hostname}
+    # shared_key <SECRET_STRING>
+
+    secure yes
+    # enable_strict_verification yes
+
+    ca_cert_path /etc/fluent/keys/ca.crt <1>
+    ca_private_key_path /etc/fluent/keys/privkey.pem <2>
```

Comment:
if you use mTLS (aka client cert auth with priv key) you will need to specify both the client cert and key using https://docs.fluentd.org/output/forward#tls_client_cert_path and https://docs.fluentd.org/output/forward#tls_client_private_key_path

Reply:
See also https://docs.fluentd.org/output/forward#how-to-connect-to-a-tls-ssl-enabled-server
that is, you can use tls without using tls_client_cert_path and tls_client_private_key_path - however, in order to use tls_client_cert_path and tls_client_private_key_path, you must also use tls